stltovoxel.py

```diff
@@ -45,12 +45,22 @@ def exportPngs(voxels, bounding_box, outputFilePath):
         path = (outputFilePattern + "%0" + size + "d.png")%height
         img.save(path)
 
+#def exportXyz(voxels, bounding_box, outputFilePath):
```

Comment:
can you remove this. I've accepted and merged a different pull request that dealt with this. 
After that I'll accept and merge.